index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module "focus-trap-react" {
+  import { Options as FocusTrapOptions } from "focus-trap";
+  import React from "react";
```

Comment:
@davidtheclark @TedDriggs 

This is incorrect for users who have `allowSyntheticDefaultImports` set to false (which is the default).

This should instead be

```jsx
import * as React from "react"
```

Reply:
Actually, there's a bit more wrong that I didn't notice until just now... When I finally got around to trying..

The export default line also breaks because it's a CommonJS export, rather than an ES6 export.

So there's one more slight fix that needs to also be made that I'll go ahead and make now in #22